html/themes/custom/common_design_subtheme/components/rw-related-content/rw-related-content.css

```diff
@@ -33,10 +31,15 @@
   }
   .rw-document + .rw-river--type-reports .rw-river__articles article {
     flex: 1 0 40%;
-    margin: 0 16px;
+    margin-right: 16px;
+    margin-left: 16px;
+  }
+  .rw-document + .rw-river--type-reports .rw-river__articles article:nth-last-child(2) {
+    padding-bottom: 0;
+    border-bottom: none;
   }
-  #main-content .rw-document + .rw-river--type-reports .rw-river__articles article:nth-child(-n+2) {
-    border-top: none;
+  .rw-document + .rw-river--type-reports .rw-river__articles article:nth-child(2) {
+    margin-top: 0; /* Compensate for cd-flow top margin */
```

Comment:
Ok, good point. I will try to consolidate those rules as they only thing unique is the selectors.

I'm not sure I understand your comment about the variants, or rather, I am not sure how I should achieve this.
For the Blog and Organizations, their River sections has the ID of `river-list` which I think is too generic (as it is also used on the pages with the Advanced search) and doesn't differentiate between the list versus card type layout. Am I missing something?

The Related Content block seems to be missing an ID which might explain why I've used `.rw-document + .rw-river--type-reports .rw-river__articles` instead of something more direct.

In the meantime, I've updated the Organizations page to have the same treatment for the cards as the Blog page.

Reply:
Hmm, for the blog and organizations, I think we can add something like the following snippet in the `reliefweb-rivers-page.html.twig` template override in the subtheme to add a class to the river list wrapper:

```
{% if river == 'organizations' or river == 'blog' %}
  {% set content = content|merge({
    '#attributes': create_attribute().addClass('rw-river--river-columns'),
  }) %}
{% endif %}
```

Then if needed in the `reliefweb-rivers-river.html.twig` template override we could add a class to the articles if the `rw-river--river-columns` (or whatever class name is chosen) is present in the `attributes`.

For the Appeals and Maps, I guess we could check on the ID.

Same for the Related content actually after adding the`#id` after the line to https://github.com/UN-OCHA/rwint9-site/blob/1ca75688bf052c135d888d0b3b9105a52a2b2f8f/html/modules/custom/reliefweb_entities/src/DocumentTrait.php#L201

What do yo think?